Browser/keywords/interaction.py

```diff
@@ -610,8 +622,8 @@ def select_options_by(
                     selector=selector, matcherJson=matchers, strict=self.strict_mode
                 )
             )
-            logger.debug(response.log)
-            return json.loads(response.json)
+        logger.debug(response.log)
+        return json.loads(response.json)

```

Comment:
`response` variable is only on valid if the `with` is valid.
Why outside `with`